ice_discharge.org

```diff
@@ -3060,7 +2735,7 @@ def filter_bad_v(v):
 vel = pd.read_csv("./tmp/dat_100_5000.csv", usecols=(lambda c: ('vel_eff' in c)))
 
 # For now, drop Sentinel1
-vel = vel[vel.columns.drop(list(vel.filter(regex='Sentinel1')))] 
+# vel = vel[vel.columns.drop(list(vel.filter(regex='Sentinel1')))] 
```

Comment:
I think `Sentinel1` is from the mapset name, which I think has been changed to `promice`. I thought we were already dropping Sentinel1 from the product, or it doesn't need to be not-dropped here (commenting out this line) because there is no Sentinel1 mapset anymore?

Reply:
The line is redudant as I have removed Sentinel1 mapset completely, I will remove it completely.